tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+  entry: ['src/index.ts'],
+  format: ['esm'],                 // Ensures proper ESM build
+  target: 'es2020',
+  dts: true,                       // Generates .d.ts files
+  sourcemap: true,
```

Comment:
```suggestion
```

We can remove both of these as consumers don't usually use types or sourcemaps from CLIs.